lib/webauto/web4cucumber.rb

```diff
@@ -965,16 +965,7 @@ def self.linux?
     end
 
     def self.container?
-      return @@container if defined?(@@container)
-
-      if File.exists? '/proc/1/cgroup'
-        cgroups = File.read '/proc/1/cgroup'
-        @@container = %w[kube docker lxc].any? { |pattern|
-          cgroups.include? pattern
-        }
-      else
-        @@container = false
-      end
+      !!ENV['KUBERNETES_SERVICE_HOST'] || !!ENV['DOCKER_CONTAINER_NAME'] || !!ENV['container']
```

Comment:
using the environment variables available for all container runtime so that we already return true correctly